Connectors/Rhino/Speckle.Connectors.GrasshopperShared/Parameters/SpeckleCollectionWrapper.cs

```diff
@@ -147,27 +147,35 @@ public int Bake(RhinoDoc doc, List<Guid> obj_ids, bool bakeObjects, int parentLa
         Guid parentLayerId = doc.Layers[parentLayerIndex].Id;
         currentLayerIndex = CreateLayer(doc, Collection.name, parentLayerId, Color);
         Guid currentLayerId = doc.Layers.FindIndex(currentLayerIndex).Id;
-        obj_ids.Add(currentLayerId);
+        objIds.Add(currentLayerId);
       }
       else
       {
-        currentLayerIndex = CreateLayerByPath(doc, Path, Color, obj_ids);
+        currentLayerIndex = CreateLayerByPath(doc, Path, Color, objIds);
       }
     }
 
     // then bake elements in this collection
     foreach (var obj in Elements)
     {
-      if (obj is SpeckleObjectWrapper so)
+      // SpeckleBlockInstanceWrapper checked before SpeckleObjectWrapper
+      if (obj is SpeckleBlockInstanceWrapper blockInstance)
       {
         if (bakeObjects)
         {
-          so.Bake(doc, obj_ids, currentLayerIndex, true);
+          blockInstance.Bake(doc, objIds, currentLayerIndex, true);
```

Comment:
are you sure this needs to be a separate block? can't this be handled in the object wrapper if bake is overridden?

Reply:
Nope. You are absolutely correct. Originally the method signatures were different. I made them uniform but forgot about simplifying this: [refactor: simplify collection baking](https://github.com/specklesystems/speckle-sharp-connectors/pull/934/commits/823cc18cd791f6ad876f3aa7ff7f1458fcb3a8de)